interface/forms/dictation/view.php

```diff
@@ -12,6 +12,8 @@
 
 use OpenEMR\Core\Header;
 include_once("../../globals.php");
+require_once("../../../public/assets/htmlpurifier-4.9.2/library/HTMLPurifier.auto.php");
```

Comment:
So the ../../../public should be replaced with which global? I looked for it and couldn't find it or remember what it is.